Mu2eG4/src/constructServicesGeom.cc

```diff
@@ -69,6 +69,8 @@ namespace mu2e {
     geomOptions->loadEntry( config, "electronicRack", "electronicRack");
     geomOptions->loadEntry( config, "pipe",           "pipe");
 
+    const bool electronicRackBuild = false;
```

Comment:
I guess this was to solve the overlap between the electronics rack that is built in the downstream STM area by this ServicesGeom and the downstream STM envelope that I made. We should handle the electronics rack in the STM area ourselves rather than use this, since this is hard-coded to have "HallAir" as a mother volume.

However, this is the wrong solution because it turns off the electronics racks for everyone. @caohc , we should turn this off in our code somewhere

Reply:
Actually, I think it doesn't appear in our code and the solution we want is to create a new ```ElectronicsRack_v02.txt``` geometry file and change this line from 1 to 0 so that we don't create the STM rack: https://github.com/Mu2e/Offline/blob/2d4131e9e4e3a70a95891439adee338577f0f121/Mu2eG4/geom/ElectronicRack_v01.txt#L10